Documentation/kubernetes/policy.rst

```diff
@@ -28,6 +28,18 @@ NetworkPolicy
 For more information, see the official `NetworkPolicy documentation
 <https://kubernetes.io/docs/concepts/services-networking/network-policies/>`_.
 
+Known missing features for Kubernetes Network Policy:
+
++---------------------------------------------------------------------------+
+| Feature                    | Tracking Issue                               |
++===========================================================================+
```

Comment:
Can you test the rendered output? I think you need plus signs above and below `|` next to `Tracking Issue`.